app/services/usps_in_person_proofing/transliterable_validator.rb

```diff
@@ -42,12 +42,21 @@ def validate(record)
         invalid_chars = get_invalid_chars(value)
         next unless invalid_chars.present?
 
+        nontransliterable_chars += invalid_chars
+
         record.errors.add(
           field,
           :nontransliterable_field,
           message: get_error_message(invalid_chars),
         )
       end
+
+      if nontransliterable_chars.present?
+        nontransliterable_chars = nontransliterable_chars.sort
+        analytics.idv_in_person_proofing_nontransliterable_characters_submitted(
+          nontransliterable_characters: nontransliterable_chars,
+        )
```

Comment:
can remove the re-assignment and inline that too?

```suggestion
        analytics.idv_in_person_proofing_nontransliterable_characters_submitted(
          nontransliterable_characters: nontransliterable_chars.sort,
        )
```

Reply:
will do